_data/projects/Spruce.yml

```diff
@@ -0,0 +1,10 @@
+name: Spruce
+desc: An open source social media platform built with Node.js and MongoDB
+site: https://github.com/DivySrivastava/spruce
+tags:
+- javascript
+- nodejs
+- mongodb
+upforgrabs:
+  name: beginner
```

Comment:
@DivySrivastava  Here, you should have the name for the label that you want to use to label your issues, such as "good first issue", "help wanted", "up-for-grabs", etc.

Reply:
@DivySrivastava you haven't yet changed name for the label. I can see that you've used good first issue label link. if you are using that then give same name here also